test/selenium/configurations.rb

```diff
@@ -36,11 +36,11 @@ def common_configuration_for_firefox_extension
   Capybara.app_host = "chrome://privly"
   @@privly_applications_folder_path = Capybara.app_host + "/content/privly-applications/"
   puts "Packaging the Firefox Extension"
-  system( "cd ../../../../../ && pwd && ./package.sh && cd chrome/content/privly-applications/test/selenium" )
+  system("cd ../../../../../ && pwd && jpm xpi && cd chrome/content/privly-applications/test/selenium")
```

Comment:
Yeah, there is - [jpmignore](https://developer.mozilla.org/en-US/Add-ons/SDK/Tools/jpmignore).
I'll check it out. The files to ignore --
```
run_test.sh
test.json
*.md
chrome/content/test - Test files
chrome/content/vendor - Jasmine
```
right ?